src/android/setup.cfg

```diff
@@ -57,7 +57,4 @@ exclude=\
     build/*
 max-complexity = 10
 max-line-length = 119
-ignore = E121,E123,E126,E226,E24,E704,W503,W504,C901
-
-[isort]
-multi_line_output = 3
```

Comment:
OK, this is probably why I had [so much trouble](https://github.com/beeware/toga/pull/1631#discussion_r1001246049) getting isort to look at the root setup.cfg file: it was using the setup.cfg files in the subdirectories instead.